projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1911,20 +1932,68 @@ private boolean loadQuestions(
       List<String> options,
       List<String> parameters,
       Map<String, String> bfq) {
-    if (!isValidArgument(options, parameters, 0, 1, 1, Command.LOAD_QUESTIONS)) {
+    if (!isValidArgument(options, parameters, 1, 0, 1, Command.LOAD_QUESTIONS)) {
       return false;
     }
-    String questionsPathStr = parameters.get(0);
-    return loadQuestions(outWriter, questionsPathStr, bfq);
+
+    boolean loadRemote = false;
+    if (options.size() == 1) {
+      if (options.get(0).equals("-loadremote")) {
+        loadRemote = true;
+      } else {
+        _logger.errorf("Unknown option: %s\n", options.get(0));
+        printUsage(Command.LOAD_QUESTIONS);
+        return false;
+      }
+    }
+    Answer answer = new Answer();
+    LoadQuestionAnswerElement ae = new LoadQuestionAnswerElement();
+    answer.addAnswerElement(ae);
+    int numLoaded = 0;
+
+    if (parameters.isEmpty() || loadRemote) {
+      //loading remote questions from batfish
+      Map<String, String> globalQuestions = _workHelper.getGlobalQuestions();
+      for (Entry<String, String> question : globalQuestions.entrySet()) {
+        int numBefore = bfq.size();
+        String name = loadQuestion(null, question.getValue(), question.getKey(), bfq);
+        int numAfter = bfq.size();
```

Comment:
This is a very indirect way to check that the question was already there. Why not `globalQuestions.hasKey(questionName)`?